Describe how to use AWS and GCP resources (#56)
capactio/website: docs/example/mattermost-installation.md

This instruction will guide you through the installation of Mattermost on a Kubernetes cluster using Capact. 

> Mattermost depends on the PostgreSQL database. Depending on the cluster configuration, with the Capact project, you can install Mattermost with a managed Cloud SQL or AWS RDS database or a locally deployed PostgreSQL Helm chart.

Reviewer: ```suggestion
Mattermost depends on the PostgreSQL database. Depending on the cluster configuration, with the Capact project, you can install Mattermost with a managed Cloud SQL, AWS RDS database or a locally deployed PostgreSQL Helm chart.
```

#### Instructions

1. Create a TypeInstance with GCP Service Account.

   1. Follow the [GCP Service Account TypeInstance creation](./typeinstances.md#gcp-service-account) guide to create and obtain ID of the newly created TypeInstance. 
   1. Export the TypeInstance ID as environment variable:

    ```bash
    export TI_ID={GCP Service Account TypeInstance ID}
    ```

>    Make sure that environment variable `TI_ID` exists.

Reviewer: ```suggestion
```

#### Instructions

> 1. Create a TypeInstance with GCP Credentials. See how to create Type Instances with [GCP credentials](typeinstances.md#gcp). When creating a new service account, assign the `Cloud SQL Admin` role to it.

Reviewer: ```suggestion
1. Create a TypeInstance with GCP Credentials. See how to create Type Instances with [GCP credentials](typeinstances.md#gcp). When creating a new service account, assign the `Cloud SQL Admin` role to it.
```

``````suggestion
1. Create a TypeInstance with GCP Service Account.

   1. Follow the [GCP Service Account TypeInstance creation](./typeinstances.md#gcp-service-account) guide to create and obtain ID of the newly created TypeInstance. 
   1. Export the TypeInstance ID as environment variable:

    ```bash
    export TI_ID={GCP Service Account TypeInstance ID}
    ```
``````

              requires:
                - path: "cap.type.gcp.auth.service-account"
            inject:
	      requiredTypeInstances:
                - id: ${TI_ID}

> 		  description: "GCP Service Account"

Reviewer: Something is broken - I think you should revert the changes in these lines:

![Screenshot 2021-09-02 at 11 32 36](https://user-images.githubusercontent.com/7155799/131820224-e4a8c7b2-edfd-42fd-8d79-ae387158d7b3.png)

helm delete -n $NAMESPACE $(helm list -f="mattermost-*" -q -n $NAMESPACE)
```

### Install Mattermost with an external AWS RDS database

To change the Mattermost installation, we need to adjust our Global policy to prefer AWS solutions. Read more about Global policy configuration [here](../feature/policies/global-policy.md).

#### Instructions

> 1. Create a TypeInstance with AWS Credentials. See how to create Type Instances with [AWS credentials](typeinstances.md#aws). 
>    Make sure that environment variable `TI_ID` exists.

Reviewer: ``````suggestion
1. Create AWS Credentials TypeInstance.

   1. Follow the [AWS Credentials TypeInstance creation](./typeinstances.md#aws-credentials) guide to create and obtain ID of the newly created TypeInstance. 
   1. Export the TypeInstance ID as environment variable:

    ```bash
    export TI_ID={AWS credentials TypeInstance ID}
    ```
``````
